### PR TITLE
Installation from Hackage - add README section

### PR DESCRIPTION
**Pull request by pepeiborra**
Following #1175

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +165,7 @@ cd haskell-language-server
 
 #### Building
 
+
```
Comment (ndmitchell) [inline README.md:168]: Why a random blank line here?

File: README.md [hunk 2]
```diff
@@ -245,6 +247,12 @@ If your desired ghc has been found, you use it to install haskell-language-serve
 ./cabal-hls-install hls-8.6.5
 ```
 
+### Installation from Hackage
+
+Direct installation from Hackage is supported via `cabal install haskell-language-server`.
+This builds the `haskell-language-server` binary and installs it in the default Cabal binaries folder.
+This binary will only work against the same GHC version that was used to build it.
```
Comment (ndmitchell) [inline README.md:254]: Maybe include the words "but not recommended"?

File: README.md [hunk 3]
```diff
@@ -245,6 +247,12 @@ If your desired ghc has been found, you use it to install haskell-language-serve
 ./cabal-hls-install hls-8.6.5
 ```
 
+### Installation from Hackage
+
+Direct installation from Hackage is supported via `cabal install haskell-language-server`.
```
Comment (ndmitchell) [inline README.md:252]:
```suggestion
Direct installation from Hackage is possible via `cabal install haskell-language-server`.
```

**Review by ndmitchell — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pepeiborra) [reply README.md:168]: It's probably inserted by the markdown VSCode plugin

File: README.md (hunk 2 above)
Comment (jneira) [reply README.md:254]:
```suggestion
This binary will only work with projects built using the same GHC version that was used to build the binary itself.
```

File: README.md (hunk 3 above)
Comment (jneira) [inline README.md:252]: you could use `stack` to install it too

Comment (pepeiborra) [reply README.md]: I don't think so, since it's not in Stackage and therefore would require a ton of extra-reps.

File: README.md (hunk 2 above)
Comment (jneira) [reply README.md]: > 
> 
> Maybe include the words "but not recommended"?

Or something like "So it is recommended for advanced users."?

File: README.md (hunk 3 above)
Comment (jneira) [reply README.md]: mmm you are right, it is possible with one command (`stack install brittany-0.13.1.0 Cabal-3.0.2.0 clock-0.7.2 ... haskell-language-server`) but not straighforward

**Review by jneira — APPROVED** [review]

**Pull request merged**